AUTHORS

```diff
@@ -3,55 +3,35 @@ many contributions from the community.
 
 Below is a list of people who contributed to the Tarantool documentation:
 
-Aleksandr Lyapunov
```

Comment:
I see that many real authors of the documentation were removed. Many past employees at least, including one that was in the documentation team.

The repository was extracted several years ago (seven or kinda) from the main repository. Some contributions are go from the main repository, but git history will not show them.

Also I would note that even if a developer does not technically create a commit, but wrote most of the content regarding a feature, he/she anyway the author and contributor.

Reply:
> <...> including one that was in the documentation team.

Ah, no. My mistake. I thought that Sofia was removed from the list.